src/main/scala/tutorial/CreateNDWIPng.scala

```diff
@@ -0,0 +1,41 @@
+package tutorial
+
+import geotrellis.raster._
+import geotrellis.raster.io.geotiff._
+import geotrellis.raster.render._
+import com.typesafe.config.ConfigFactory
+
+
+/**
+  * Created by FroehlingGallier on 10/13/16.
+  */
+object CreateNDWIPng {
+
+  val maskedPath = "data/r-g-nir.tif"
+  val ndviPath = "data/ndwi.png"
+
+  def main(args: Array[String]): Unit = {
+    val ndwi = {
+      // Convert the tile to type double values,
+      // because we will be performing an operation that
+      // produces floating point values.
+      println("Reading in multiband image...")
+      val tile = MultibandGeoTiff(maskedPath).convert(DoubleConstantNoDataCellType)
+
+      // Use the combineDouble method to map over the red and infrared values
+      // and perform the NDVI calculation.
+      println("Performing NDWI calculation...")
+      tile.combineDouble(1, 2) { (g: Double, ir: Double) =>
+        Calculations.ndwi(g, ir);
+      }
+    }
+
+    // Get color map from the application.conf settings file.
+    val colorMap = ColorMap.fromStringDouble(ConfigFactory.load().getString("tutorial.colormap")).get
```

Comment:
We should have dedicated color map for NDWI.

`0:aacdff44;0.1:70abffff;0.2:3086ffff;0.3:1269e2ff;0.4:094aa5ff;1:012c69ff`

then we can have `tutorial.ndviColormap` and `tutorial.ndwiColormap`